### ISSUE TITLE
No attribute when it should exist

### ISSUE THREAD

**Issue by debedb** [mk-fg/layered-yaml-attrdict-config#9]
Consider YAML like this:

```
boo:
  - one: a
    x : z
  - two: x
```

And code parsing it

```
config = lya.AttrDict.from_yaml(file)
print config.boo[0].x
```

which gives an error

```
Traceback (most recent call last):
  File "config.py", line 33, in <module>
    main()
  File "config.py", line 22, in main
    print config.boo[0].x
AttributeError: 'OrderedDict' object has no attribute 'x'
```

**Comment by mk-fg** [comment]
There's a reason this module doesn't go into lists, as mentioned in #6, but I think it'd be indeed nice (and won't hurt) to have attribute access for these, maybe also with some callable parameter that'd define how to merge these.

So yeah, definitely something to be fixed, thanks for mentionng it.

**Comment by mk-fg** [comment]
Should be fixed as of ea12fbf and 15.8.1 on PyPI.

See also this note: https://github.com/mk-fg/layered-yaml-attrdict-config/#lists-and-tuples-inside-attrdicts
